services/QuillLMS/engines/evidence/Gemfile.lock

```diff
@@ -65,37 +66,41 @@ GEM
     factory_bot_rails (4.8.2)
       factory_bot (~> 4.8.2)
       railties (>= 3.0.0)
-    faraday (1.8.0)
+    faraday (1.10.0)
       faraday-em_http (~> 1.0)
       faraday-em_synchrony (~> 1.0)
       faraday-excon (~> 1.1)
-      faraday-httpclient (~> 1.0.1)
+      faraday-httpclient (~> 1.0)
+      faraday-multipart (~> 1.0)
       faraday-net_http (~> 1.0)
-      faraday-net_http_persistent (~> 1.1)
+      faraday-net_http_persistent (~> 1.0)
       faraday-patron (~> 1.0)
       faraday-rack (~> 1.0)
-      multipart-post (>= 1.2, < 3)
+      faraday-retry (~> 1.0)
       ruby2_keywords (>= 0.0.4)
     faraday-em_http (1.0.0)
     faraday-em_synchrony (1.0.0)
     faraday-excon (1.1.0)
     faraday-httpclient (1.0.1)
+    faraday-multipart (1.0.3)
+      multipart-post (>= 1.2, < 3)
     faraday-net_http (1.0.1)
     faraday-net_http_persistent (1.2.0)
     faraday-patron (1.0.0)
     faraday-rack (1.0.0)
+    faraday-retry (1.0.3)
     ffi (1.15.5)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    gapic-common (0.7.0)
+    gapic-common (0.8.0)
       faraday (~> 1.3)
       google-protobuf (~> 3.14)
       googleapis-common-protos (>= 1.3.11, < 2.a)
       googleapis-common-protos-types (>= 1.0.6, < 2.a)
       googleauth (>= 0.17.0, < 2.a)
       grpc (~> 1.36)
-    globalid (0.5.2)
+    globalid (1.0.0)
```

Comment:
This major version bump raised a flag for me, but I checked the changeset, and the 'major' version bump was cosmetic: https://github.com/rails/globalid/releases/tag/v1.0.0

Reply:
Thanks for confirming.